pkg/tag/tag.go

```diff
@@ -84,15 +84,20 @@ func (t Tags) Contains(item *Tag) bool {
 type Info struct {
 	Tag Tag
 	// Data encoding "UL", "CS", etc.
+	// For tags with multiple allowed VRs, they are separated as "VR1 or VR2 or ... or VRn".
 	VR string
```

Comment:
Should we make this a `[]string` now that we consider all possible VRs accepted?